pkg/pillar/containerd/containerd.go

```diff
@@ -596,3 +693,42 @@ func createMountPointExecEnvFiles(containerPath string, mountpoints map[string]s
 
 	return nil
 }
+
+// LKTaskLaunch runs a task in a new containter created as per linuxkit runtime OCI spec
+// file and optional bundle of DomainConfig settings and command line options. Because
+// we're expecting a linuxkit produced filesystem layout we expect R/O portion of the
+// filesystem to be available under `dirname specFile`/lower and we will be mounting
+// it R/O into the container. On top of that
```

Comment:
Did the comment get truncated?

Reply:
Yup -- it seems to be the same rebase that ate up that Kill related line. Thanks for the catch (lesson here: I should avoid rebasing after hacking for a long time ;-))